parser/openfoodfacts_taxonomy_parser/utils.py

```diff
@@ -4,7 +4,7 @@
 import unidecode
 
 
-def normalize_text(line: str, lang="default", char="-"):
+def normalize_text(line: str, lang="default", char="-", stopwords={}):
```

Comment:
```suggestion
def normalize_text(line: str, lang: str="default", char: str="-", stopwords: dict[str, str] | None=None) -> str:
   if stopwords is None:
      stopwords={}
```
https://realpython.com/python-optional-arguments/#data-types-that-shouldnt-be-used-as-default-arguments

Reply:
Well spotted.